exercises/D-strings/exercise.js

```diff
@@ -1,3 +1,5 @@
 // Start by creating a variable `message`
-
+let message="Tony was here";
```

Comment:
In order to make your code is super clear to read, it's important to stick to a few conventions when you're writing variable declarations like this. You should have spaces between the variable name `message` and the assignment operator `=` like this:
```suggestion
let message = "Tony was here";
```